src/Sulu/Bundle/AdminBundle/Controller/AdminController.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
@alexander-schranz i think this would be a BC break?

Reply:
reverted it anyway, as we do not need backend adjustments with the new solution